actions/pr-slack-alert/payload.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Seems like this payload will create a message for reviewing a PR, would it make sense to generify it based on the pr event type?
Doesn't need to be done now, could maybe add a section in the readme for future work.

Reply:
ahh good call out!